chef/cookbooks/crowbar/templates/default/crowbar-shutdown-services-before-upgrade.sh.erb

```diff
@@ -64,4 +72,13 @@ for service in cinder-volume cinder-scheduler; do
     done
 done
 
+ret=$?
+if [ $ret != 0 ] ; then
+    echo "Error occured during deleting cinder services"
+    echo $ret > $UPGRADEDIR/crowbar-shutdown-services-before-upgrade-failed
```

Comment:
Nothing is removing this file on the script. Shouldn't we try to remove it at the start of it? If not, that means that if it fails one, it will be reported as failing forever (If the api uses this file to report the status)

If it doesnt use it, then Im not sure the point of the file?